lab/user-manual.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Memgraph Lab user manual
+sidebar_label: User manual
+---
+
+**Memgraph Lab** is a **visual user interface** that enables you to import and
+export data to and from Memgraph database, write and execute Cypher queries,
+visualize graph data, view and optimize query performance, develop query modules
+in Python or connect to data streams. 
+
+Here is a short overview of the Lab's interface, features it provides, and links
+to resources that will help you achieve your graph goals. 
+
+## Overview
+
+Every time you open Memgraph Lab, it will greet you with an **Overview**,
+offering resources and actions depending whether your database is empty or not,
+and whether you ran any queries or not.
+
+<img src={require('./data/user-manual/overview.png').default} className={"imgBorder"} style={{marginBottom:'20px'}}/>
+
+At the top of the screen, you will find information about: 
+- Connection status
+- Memgraph version, IP address, and port of the database
+- Number of nodes and relationships currently in the database
+- Disk storage used, and total and available RAM 
+
+In the top right corner, you can find the help button that will provide you with
+helpful documentation and links to the Memgraph community.
+
+All the Memgraph Lab sections are listed in the left side menu, below which you
+can find the [Layout](#layout) options and Memgraph Lab version. 
+
+## Query Execution
+
+In this section, you can write and run queries, as well as see their tabular or
+graphical results. 
+
+### Cypher Editor
+
+Here is where you write and run your Cypher queries. A keyword suggestion tool
+can help you with clause completion and give information about signatures and
+parameters. If you need help writing Cypher queries, check out the [Cypher
+manual](/cypher-manual).
+
+<img src={require('./data/user-manual/intelisense.png').default} className={"imgBorder"} style={{marginBottom:'20px'}}/>
+
+Once you **Run** a query (by clicking a button or pressing **CTRL** +
+**Enter**), you can **Cancel** it, but if the query has already
+reach MemgraphDB, the action won't actually stop the query's execution. In the
+case of running complex algorithms on a large dataset, you need to be a bit
+patient and wait for Memgraph to complete running the query. 
+
+When the query has executed or failed, below the Cypher editor, you can see a

```

Comment:
It would maybe make sense to have image showing this here or maybe move this part to the Data results section?